helios-client/src/main/java/com/spotify/helios/client/HeliosClient.java

```diff
@@ -576,6 +577,24 @@ public Builder setShutDownExecutorOnClose(final boolean shutDownExecutorOnClose)
       return this;
     }
 
+    /**
+     * Set the per-request HTTP connect/read timeout used when communicating with master. Default is
+     * 10 seconds.
+     */
+    public Builder setHttpTimeout(final int timeout, TimeUnit unit) {
```

Comment:
because [`URLConnection.setConnectTimeout`](https://docs.oracle.com/javase/7/docs/api/java/net/URLConnection.html#setConnectTimeout(int)) and `setReadTimeout` only take ints